automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,468 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
 
   "A workspace with an authorization domain" - {
+    "with one group inside of it" - {
+      "can be created" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { authDomainName =>
+          withCleanUp {
+            val workspaceListPage = signIn(Config.Users.fred)
 
-    "can be created by a user who is in a managed group" in withWebDriver { implicit driver =>
-      withGroup("AuthDomainSpec") { authDomainName =>
-        withCleanUp {
-          val workspaceListPage = signIn(Config.Users.fred)
-
-          val workspaceName = "AuthDomainSpec_create_" + randomUuid
-          register cleanUp api.workspaces.delete(projectName, workspaceName)
-          val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Option(authDomainName)).awaitLoaded()
+            val workspaceName = "AuthDomainSpec_create_" + randomUuid
+            register cleanUp api.workspaces.delete(projectName, workspaceName)
+            val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Set(authDomainName)).awaitLoaded()
 
-          workspaceDetailPage.ui.readAuthDomainGroups should include (authDomainName)
+            workspaceDetailPage.ui.readAuthDomainGroups should include(authDomainName)
 
-          workspaceListPage.open.filter(workspaceName)
-          workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+            workspaceListPage.open.filter(workspaceName)
+            workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
```

Comment:
I definitely understand why. I do worry about the potential explosion of test cases that this split up could cause. `looksRestricted` is used in 14 test cases in `AuthDomainSpec`.

Maybe not all of those need to be there. Maybe we could have one test that reads the auth domain groups on the summary page, one that checks that it looks restricted on the list page, then the remaining cases that assert based on access/visibility and don't actually "read" the view like the first 2.